cfme/tests/configure/test_access_control.py

```diff
@@ -1446,7 +1445,24 @@ def test_tenant_unique_catalog():
         initialEstimate: 1/2h
         startsin: 5.5
     """
-    pass
+    catalog_name = fauxfactory.gen_alphanumeric()
+    catalog_desc = "My Catalog"
+    msg = "Name has already been taken"
+
+    cat1 = appliance.collections.catalogs.create(name=catalog_name, description=catalog_desc)
```

Comment:
Optional:
As one catalog is setup for this test. I would personally move it as fixture. So in future if someone want to use catalog then he/she can directly use.